manifests/master/bmc_smtp.pp

```diff
@@ -7,24 +7,33 @@
 
   include ::xinetd
 
-  # Get IPMI network CIDR
-  $ipmi_networks = lookup( 'profile_xcat::ipmi_net_cidrs', Array )
+  #Check for defined bind address
+  $ipmi_bind_ip = lookup( 'profile_xcat::ipmi_bind_ip', String, undef, 'Null' )
 
-  $ipmi_networks.each | $cidr | {
-    # Validate proper network address for IPMI network
-    $tgt_net = ip_address( ip_network( $cidr ) )
+  #If defined set the bind IP and verify the correct IP address
+  if ( $ipmi_bind_ip != 'Null' ) {
```

Comment:
change this to compare for `undef` (required with changes above)

Reply:
Fixed